core/src/main/java/com/netflix/conductor/metrics/Monitors.java

```diff
@@ -306,7 +306,19 @@ public static void recordAcquireLockFailure(String exceptionClassName) {
 		counter(classQualifier, "acquire_lock_failure", "exceptionType", exceptionClassName);
 	}
 
-	public static void recordSystemTaskWorkerPollingLimited(String queueName) {
+	public static void recordWorkflowCompleted(String workflowType) {
+		counter(classQualifier, "workflow_completed", "workflowName", workflowType);
+	}
+
+	public static void recordTaskCompleted(String taskType) {
+		counter(classQualifier, "task_completed", "taskType", taskType);
+	}
+
+	public static void recordSystemTaskCompleted(String taskType) {
```

Comment:
How about adding tags to `task_completed` metric instead?

Reply:
Adding tasks as separate tags will be beneficial for aggregation purposes as well.